compiler/noirc_driver/src/program.rs

```diff
@@ -29,4 +29,6 @@ pub struct CompiledProgram {
     pub warnings: Vec<SsaReport>,
     /// Names of the functions in the program. These are used for more informative debugging and benchmarking.
     pub names: Vec<String>,
+    /// Names of the unconstrained functions in the program.
+    pub brillig_names: Vec<String>,
```

Comment:
What is the ordering in this Vec? How do we know which name is which?

Reply:
They are fetched in `compiler/noirc_evaluator/src/ssa/acir_gen/mod.rs` in the same order that the brillig bytecode is collected. I just pushed a small update as well to improve this collection. We do a similar process into of `ssa.rs` for circuit names as we want to separate out the debug information from the actual bytecode artifact.